addons/account/report/account_invoice_report_view.xml

```diff
@@ -101,15 +101,15 @@
         <field name="name">Invoices Analysis</field>
         <field name="res_model">account.invoice.report</field>
         <field name="view_mode">graph,pivot</field>
-        <field name="context">{'search_default_current':1, 'search_default_supplier':1, 'group_by':[], 'group_by_no_leaf':1, 'search_default_year': 1}</field>
+        <field name="context">{'search_default_current':1, 'res_partner_search_mode': 'supplier', 'group_by':[], 'group_by_no_leaf':1, 'search_default_year': 1}</field>
```

Comment:
This only works if you have `sale` and `purchase` installed. The concept of customer/supplier should depend on those modules.